src/main/scenes/svg.tsx

```diff
@@ -86,28 +86,12 @@ const getInitialState = ({ model, options }: Props): State => {
       : [];
 
     element.deserialize(apollonElement, apollonChildren);
-    const children: UMLElement[] = apollonChildren.reduce<UMLElement[]>(
-      (acc, val) => [...acc, ...deserialize(val)],
-      [],
-    );
-
-    const [root, ...updates] = element.render(layer, children) as UMLElement[];
-    updates.map((x) => {
-      const original = apollonChildren.find((y) => y.id === x.id);
-      if (!original) {
-        return x;
-      }
-      x.bounds.x = original.bounds.x;
-      x.bounds.y = original.bounds.y;
-      return x;
-    });
+    const [root, ...updates] = element.render(layer, []) as UMLElement[];
 
     return [root, ...updates];
```

Comment:
I guess this could simplify to `return element.render(layer, []) as UMLElement[];` but it's not really important.